From 298826dd718fd48f564170e727fecfc2dcd94118 Mon Sep 17 00:00:00 2001
From: Adam Duskett <adam.duskett@amarulasolutions.com>
Date: Thu, 9 Nov 2023 14:38:03 -0700
Subject: [PATCH] test

Signed-off-by: Adam Duskett <adam.duskett@amarulasolutions.com>
---
 src/install/dracut-install.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/install/dracut-install.c b/src/install/dracut-install.c
index 96b20e9..88f68e8 100644
--- a/src/install/dracut-install.c
+++ b/src/install/dracut-install.c
@@ -333,7 +333,7 @@ normal_copy:
                                     && no_xattr == false) ? "--preserve=mode,xattr,timestamps,ownership" : "--preserve=mode,timestamps,ownership";
         if (pid == 0) {
                 execlp("cp", "cp", "--reflink=auto", "--sparse=auto", preservation, "-fL", src, dst, NULL);
-                _exit(errno == ENOENT ? 127 : 126);
+                _exit(errno == 126);
         }
 
         while (waitpid(pid, &ret, 0) == -1) {
@@ -539,7 +539,7 @@ static int resolve_deps(const char *src)
                 dup2(fds[1], 2);
                 putenv("LC_ALL=C");
                 execlp(ldd, ldd, fullsrcpath, (char *)NULL);
-                _exit(errno == ENOENT ? 127 : 126);
+                _exit(errno == 126);
         }
         close(fds[1]);
 
-- 
2.41.0

